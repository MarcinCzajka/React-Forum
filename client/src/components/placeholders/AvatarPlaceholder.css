.avatarPlaceholderContainer {
    display: block;
    float: left;
    background-color: #EFF3F4;
    margin-right: 1em;
    line-height: 1;
}

.avatarPlaceholder {
    display: block !important;
    margin: 0 !important;
    color: #618993 !important;
    font-size: 2.5em !important;
    width: 1em !important;
}