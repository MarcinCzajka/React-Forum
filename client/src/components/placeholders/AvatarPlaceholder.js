import React from 'react';
import { Icon } from "semantic-ui-react";
import './AvatarPlaceholder.css';

class AvatarPlaceholder extends React.Component {

    render() {
        return (
            <div className={`avatarPlaceholderContainer ${this.props.size}`} >
                <Icon className='avatarPlaceholder' name='user' />
            </div>
        )
    }

}

export default AvatarPlaceholder;