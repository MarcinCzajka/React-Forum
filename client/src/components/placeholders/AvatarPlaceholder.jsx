import React from 'react';
import { Icon } from "semantic-ui-react";
import './AvatarPlaceholder.scss';

export default props => {
    return (
        <div className={`avatarPlaceholderContainer ${props.size}`} >
            <Icon className='avatarPlaceholder' name='user' />
        </div>
    )
}