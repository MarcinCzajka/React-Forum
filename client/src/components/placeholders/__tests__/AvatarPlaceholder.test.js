import React from 'react';
import { create } from "react-test-renderer";
import AvatarPlaceholder from '../AvatarPlaceholder';

describe('Captcha component', () => {
    test('match snapshot', () => {
        const component = create(<AvatarPlaceholder />);
        expect(component).toMatchSnapshot();
    });

});