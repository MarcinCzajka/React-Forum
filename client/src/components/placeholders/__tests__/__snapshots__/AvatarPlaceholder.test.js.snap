// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Captcha component match snapshot 1`] = `
<div
  className="avatarPlaceholderContainer undefined"
>
  <i
    aria-hidden="true"
    className="user icon avatarPlaceholder"
    onClick={[Function]}
  />
</div>
`;
